tests/integration/docker-compose.yml

```diff
@@ -16,18 +16,40 @@ services:
       - LOGOS_ORIGIN=null
       - BREACH_RESOLUTION_ENABLED=false
       - MONITOR_FXA_PASSWORD
+      - DATABASE_URL=postgres://postgres@postgres:5432/blurts
```

Comment:
I was curious if you could recycle the environment's `DATABASE_URL` variable, but i think this is subtlety different than the .env-dist file above, which has:

```ini
DATABASE_URL="postgres://postgres@localhost:5432/blurts"
```
(an `@localhost:5432` instead of `@postgres:5432`).

Reply:
Yes different because docker-compose. Using ```@postgres``` points to the postgres container